playbooks/roles/setup-users/tasks/main.yml

```diff
@@ -1,4 +1,4 @@
 ---
```

Comment:
now that we are no longer installing wetty on the masters, do we even need to install the users? The users were created so they could access the master via wetty, correct? With butterfly there are no logins, so i think users are no longer necessary..

Reply:
@ecwpz91  actually would deleting removing users task break the oscap lab? I think quay doesn't need it but not so sure about oscap